pkg/server/plugin/notifier/v1.go

```diff
@@ -45,39 +44,3 @@ func (v1 *V1) NotifyBundleUpdated(ctx context.Context, bundle *common.Bundle) er
 	})
 	return v1.WrapErr(err)
 }
-
-func bundleFromCommonProto(b *common.Bundle) (*types.Bundle, error) {
```

Comment:
This has been replaced with the bundle.ToPluginProtoFromCommon function, which now performs validations during the conversion in the same way that other  conversion functions do. Because of this, updating the test data of unit tests depending on this function was also needed.